src/features/presence/PresenceDropdown.js

```diff
@@ -97,12 +99,17 @@ class PresenceDropdown extends React.Component<Props> {
 
     renderTimestampMessage = (interactedAt, interactionType) => {
         const lastActionMessage = determineInteractionMessage(interactionType, interactedAt);
+        const { intl } = this.props;
+        const timeAgo = intl.formatRelativeTime
```

Comment:
Uses new version if available, otherwise old version.
Need to double check if the output is correct, specially the delta.